larray/tests/test_array.py

```diff
@@ -131,6 +155,19 @@ def test_getattr(self):
         with self.assertRaises(AttributeError):
             self.larray.geom
 
+    @pytest.mark.skipif(PY2, reason="Python 2.7")
```

Comment:
The reason is not clear enough. I don't understand why it needs to be skipped on python 2.7... Ah, maybe it's because of the broken `__eq__`. If this is the reason, wouldn't it be better to fix `__eq__` than skip the test?

Reply:
The reason is that all of our current users have Python 3 on their computers and Python 2 will be abandon soon. Implementing an AttributeDict class in Python 3 is easy while the implementation for Python 2 is ugly. My first plan was to not implement attributes for Python 2 or using a simple OrderedDict.

Reply:
We will drop Python2 as soon as possible. Until then, it would make support (of LIAM2) harder if we have different functionality on Python2.